### PR TITLE
Remove unused cp features

### PR DESCRIPTION
**Pull request by roshankern**
This PR is ready for review!

A quick PR to make sure that the only features compiled into the final labeled single-cell dataframe are those that we want to train with. These changes all take place in `utils/load_utils.py`. Before this PR, some location features were included that did not help the final model accuracy (as they are related to the cell's specific location).

This PR has about ~30 lines to review.

### CODE REVIEW

File: 3.normalize_data/normalize_data.py [hunk 1]
```diff
@@ -37,21 +39,24 @@
...
 )
+print("Loading positive control data...")
 data = compile_mitocheck_batch_data(data_path)
-print("Normalizing")
+print("Normalizing positive control data...")
 normalized_data = get_normalized_mitocheck_data(data, normalization_scaler)
 # save normalized positive control data
-print("Saving")
 save_path = pathlib.Path(f"{results_dir}/positive_control_data.csv.gz")
+print("Saving normalized positive control data...")
```
Comment (MikeLippincott) [inline 3.normalize_data/normalize_data.py:65]: I like the increased specificity in the documentation.

File: 3.normalize_data/normalize_data.py [hunk 2]
```diff
@@ -37,21 +39,24 @@
...
+print("Normalizing negative control data...")
 normalized_data = get_normalized_mitocheck_data(data, normalization_scaler)
 # save normalized negative control data
 save_path = pathlib.Path(f"{results_dir}/negative_control_data.csv.gz")
+print("Saving normalized negative control data...")
 normalized_data.to_csv(save_path, compression="gzip", index=False)
 
 # normalize positive control data
-print("Loading")
 data_path = pathlib.Path(
```
Comment (MikeLippincott) [inline 3.normalize_data/normalize_data.py]: consider declaring paths at beginning and changing path name variable to be more descriptive
```suggestion
positive_control_data_path = pathlib.Path(
```

File: utils/load_utils.py [hunk 3]
```diff
@@ -35,13 +35,28 @@ def compile_mitocheck_batch_data(
...
+    # remove CP columns that dont have a feature module as a substring
+    for col in cols_to_load:
+        if "CP__" not in col:
+            continue
+        has_feature_module = any(feature_module in col for feature_module in cp_feature_modules)
+        if not has_feature_module:
+            cols_to_remove.append(col)
+    
+    # remove columns we don't want from the list to load
+    cols_to_load = list(set(cols_to_load) - set(cols_to_remove))
```
Comment (MikeLippincott) [inline utils/load_utils.py:54]: I like this conditional with the `'CP_and_DP'` default arg passed through.

**Review by MikeLippincott — APPROVED** [review]
One major comment: consider defining paths at top of file, unless I missed a dynamic assignment of the paths then please ignore. LGTM after you address comments!

File: 3.normalize_data/normalize_data.py (hunk 2 above)
Comment (roshankern) [reply 3.normalize_data/normalize_data.py]: Done in https://github.com/WayScience/mitocheck_data/pull/35/commits/653872344885dcc3b2a0f4ce24bca87a444a65c5

**Pull request merged**